rest/src/main/kotlin/com/gitlab/kordlib/rest/builder/message/EmbedBuilder.kt

```diff
@@ -14,6 +14,10 @@ import java.time.format.DateTimeFormatter
  */
 @KordDsl
 class EmbedBuilder : RequestBuilder<EmbedRequest> {
+    companion object {
+        val ZERO_WIDTH_SPACE = "\u200E"
+    }
+
```

Comment:
1) can be made a const.

2) Wouldn't this fit better in the `Field` class, as a non-companion value?

I'm thinking in lieu of ease of use and API discovery changing this
```kotlin
field {
    name = title
    value = if(condition) text else EmbedBuilder.ZERO_WIDTH_SPACE
}
```
to this
```kotlin
field {
    name = title
    value = if(condition) text else ZERO_WIDTH_SPACE
}
```
might be beneficial.